sdk/python/foundation-models/azure_openai/openai_completions_finetune_pipeline.ipynb

```diff
@@ -0,0 +1,285 @@
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "7a744987",
+   "metadata": {},
+   "source": [
+    "## 2.2 Inspect loaded components"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17744c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Pipeline component loaded successfully. Component spec:\")\n",
+    "print(openai_completions_finetune_component)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "cb7335d1",
+   "metadata": {},
+   "source": [
+    "# 3. Sample pipeline job\n",
+    "## 3.1 Build pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "0246232c-dc7b-4d9a-b9c4-de43782121ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Construct pipeline\n",
+    "@pipeline()\n",
+    "def pipeline_with_registered_components(\n",
+    "    train_dataset, validation_dataset, training_max_epochs=20, model=\"ada\"\n",
+    "):\n",
+    "    openai_completions_finetune_component_results = (\n",
+    "        openai_completions_finetune_component(\n",
+    "            train_dataset=train_dataset,\n",
+    "            validation_dataset=validation_dataset,\n",
+    "            n_epochs=training_max_epochs,\n",
+    "            model=model,\n",
+    "            registered_model_name=\"ft_ada_model_from_sdk\",\n",
+    "        )\n",
+    "    )\n",
+    "\n",
+    "    return openai_completions_finetune_component_results\n",
+    "\n",
+    "\n",
+    "pipeline_job = pipeline_with_registered_components(\n",
+    "    train_dataset=Input(type=\"uri_folder\", path=\"data/\"),\n",
+    "    validation_dataset=Input(type=\"uri_folder\", path=\"data/\"),\n",
+    "    training_max_epochs=1,\n",
+    "    model=\"ada\",  # Select any model from [\"ada\", \"babbage\", \"curie\", \"davinci\", \"text-davinci-fine-tune-002\"]\n",
+    ")\n",
+    "\n",
+    "# set pipeline level compute\n",
+    "pipeline_job.settings.default_compute = \"serverless\"\n",
+    "pipeline_job.settings.force_rerun = True"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "811cddb4",
+   "metadata": {},
+   "source": [
+    "## 3.2 Configure workspace details and get a handle to the workspace\n",
+    "\n",
+    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](https://github.com/Azure/azureml-examples/blob/6142c51451561447befa665e8dd6fb3ff80bdb62/sdk/python/jobs/configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
```

Comment:
The same here, let us say about default authentication at the begin of the notebook.

Reply:
Adressed